.github/workflows/pull-request.yml

```diff
@@ -19,6 +19,8 @@ env:
   SCREENSHOT_PASSWORD: ${{ secrets.SCREENSHOT_PASSWORD}}
   PLAYWRIGHT_E2E_HOST: ${{ secrets.PLAYWRIGHT_E2E_HOST }}
   GH_BUILD_NUMBER: ${{ github.run_id }}
+  REPORT_PORTAL_URL: ${{ secrets.REPORT_PORTAL_URL }}
```

Comment:
Are those secrets available for ACA?

Reply:
yes, its working , I can see results on report portal. https://reportportal.envalfresco.com/ui/#alfresco-content-app/dashboard/472